.github/workflows/release-containers.yml

```diff
@@ -60,113 +89,183 @@ jobs:
-        set -- $PKG
-        echo "TAR_NAME=$1" >> $GITHUB_ENV
-        export PKG=$(ls arcade/dist/ | grep whl)
-        set -- $PKG
-        echo "WHL_NAME=$1" >> $GITHUB_ENV
+        make publish-ghcr VERSION=${{ needs.set-version.outputs.version }} ARCH=${{ matrix.arch }}
 
 
-    - name: Generate release notes
-      id: generate_release_notes
-      if: github.event_name != 'push' || startsWith(github.ref, 'refs/tags/')
-      run: |
-        echo "Release notes for version ${{ env.VERSION }}" > release_notes.md
-        echo "" >> release_notes.md
-        echo "Changes in this release:" >> release_notes.md
-        git log $(git describe --tags --abbrev=0)..HEAD --pretty=format:"- %s" >> release_notes.md
-
-    - name: Create Release
-      id: create_release
-      if: github.event_name != 'push' || startsWith(github.ref, 'refs/tags/')
-      uses: actions/create-release@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        tag_name: ${{ env.VERSION }}
-        release_name: Release ${{ env.VERSION }}
-        body_path: release_notes.md
-        draft: false
-        prerelease: false
-
-    - name: Upload Release Asset
-      uses: actions/upload-release-asset@v1
-      if: github.event_name != 'push' || startsWith(github.ref, 'refs/tags/')
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        upload_url: ${{ steps.create_release.outputs.upload_url }}
-        asset_path: ./release_notes.md
-        asset_name: release_notes.md
-        asset_content_type: text/markdown
-
-    - name: Upload Python Tar Asset
-      uses: actions/upload-release-asset@v1
-      if: github.event_name != 'push' || startsWith(github.ref, 'refs/tags/')
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        upload_url: ${{ steps.create_release.outputs.upload_url }}
-        asset_path: ./arcade/dist/${{ env.TAR_NAME }}
-        asset_name: ${{ env.TAR_NAME }}
-        asset_content_type: application/zip
-
-    - name: Upload Python Whl Asset
-      uses: actions/upload-release-asset@v1
-      if: github.event_name != 'push' || startsWith(github.ref, 'refs/tags/')
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        upload_url: ${{ steps.create_release.outputs.upload_url }}
-        asset_path: ./arcade/dist/${{ env.WHL_NAME }}
-        asset_name: ${{ env.WHL_NAME }}
-        asset_content_type: application/zip
+  push-manifest:
+    runs-on: ubuntu-latest
+    needs: [set-version, build-and-push]
+    permissions:
+      contents: write
+      packages: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
-    - name: Zip Full Dist
-      if: github.event_name != 'push' || startsWith(github.ref, 'refs/tags/')
-      run: tar -cvf python-package-distributions.tar.gz ./dist
+      - name: Log in to GitHub Container Registry
```

Comment:
there was already a step for logging in to GHCR above

Reply:
Thats in a different job, so I have to readd it since they technically run on different machines